db-scripts/src/main/resources/cgds.sql

```diff
@@ -407,6 +408,18 @@ CREATE TABLE `mutation_count` (
 );
 
 -- --------------------------------------------------------
+CREATE TABLE `mutation_variant_count` (
+    `GENETIC_PROFILE_ID` int(11) NOT NULL,
+    `KEYWORD` varchar(50) DEFAULT NULL,
```

Comment:
Thinking of matching to position as well, maybe instead of using `KEYWORD`, we could use `MUTATION_EVENT_ID`? And we can put `COUNT_BY_PROTEIN_POSITION` as a placeholder now. Later we can even extended it to `COUNT_BY_NT_POSITION` (which is actually asked by our users).